bbb-exporter/collector.py

```diff
@@ -66,6 +81,15 @@ def collect(self):
         bbb_meetings_video_participants.add_metric([], no_video_participants)
         yield bbb_meetings_video_participants
 
+
+        bbb_meetings_participant_clients = GaugeMetricFamily('bbb_meetings_participants_clients',
+                                                            "Total number of participants in all BigBlueButton "
+                                                            "meetings by client",
+                                                            labels=["clientType"])
+        for client, num in participants_by_client.items():
+            bbb_meetings_participant_clients.add_metric([client], num)
```

Comment:
I would use lowercase for the label's value, just to be consistent with metrics conventions. You can just call `client.lower()` here to accomplish that.